Explorer/Assets/DCL/Character/CharacterMotion/Systems/RotateCharacterSystem.cs

```diff
@@ -51,5 +53,18 @@ protected override void Update(float t)
             // If we are on a platform we save our local rotation
             PlatformSaveLocalRotation.Execute(ref platformComponent, characterTransform.forward);
         }
+
+        [Query]
+        private void ForceLookAt(in Entity entity, ref CharacterRigidTransform rigidTransform, ref CharacterTransform transform, in PlayerLookAtIntent lookAtIntent)
+        {
+            // Rotate player to look at camera target
+            Vector3 newLookDirection = lookAtIntent.LookAtTarget - transform.Position;
+            newLookDirection.y = rigidTransform.LookDirection.y;
+            newLookDirection.Normalize();
+            rigidTransform.LookDirection = newLookDirection;
+            transform.Transform.forward = newLookDirection;
```

Comment:
There is no need to force the rotation into the new value, this system is going to lerp the rotation properly

Reply:
@Kinerius I forced the rotation of the transform, apart from setting the `rigidTransform.LookDirection`, because I wanted to keep exactly the same behavior as the old project, where the avatar is rotated to the indicated direction instantaneously and not with a lerp. I have just tried not to force the rotation and, effectively, it works, but the avatar spent a little time to have the new orientation instead of appearing directly looking to the final direction.

In my opinion, in order to avoid possible issues with already implemented features in some scenes, we should respect exactly  the old behvior, but let me know what do you think, i don't midn to change it! :)

Reply:
I didn't knew the use case of this action, so if its meant to be instant so be it 👍